tools/cloud-build/daily-tests/ansible_playbooks/multigroup-integration-test.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: "Multigroup integration test for ghpc deploy command"
+  hosts: localhost
+  force_handlers: true
+  tasks:
+  - name: Create Deployment Directory
+    ansible.builtin.include_tasks:
+      file: tasks/create_deployment_directory.yml
+  - name: Deploy from deployment directory
+    block:
+    - name: Execute ghpc deploy
+      register: deployment
+      changed_when: deployment.changed
+      ansible.builtin.command: ./ghpc deploy {{ deployment_name }} --auto-approve
+      args:
+        chdir: "{{ workspace }}"
+      environment:
+        TF_IN_AUTOMATION: "TRUE"
```

Comment:
It would have been great if you could make this call the slurm integration tests from here, as it could test that the cluster actually worked.

Reply:
I want to add that functionality in the next update to these tests, which intention of further reducing code duplication. The approach of getting the login node IP (gcloud command piped to a regexp search) is fragile and broke recently and I don't want to expand its radius until we resolve it. I realzie that the task I captured for that work is related but doesn't capture it fully so I will update the task.

(The more solid solution is to get the IP/DNS name from `terraform output`).